recipes/pyalveo/meta.yaml

```diff
@@ -0,0 +1,74 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "pyalveo" %}
+{% set version = "1.0.3" %}
+{% set sha256 = "35cbdeb6423bfb6f9a2fb72d7707fdfe17a69fe0eda9a514c73b23604809c5fc" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
```

Comment:
It looks like `python` and `setuptools` are the only `build` requirements. Typically those are all you need if you're using `--single-version-externally-managed`, unless the `setup.py` imports them directly or indirectly or if there is a `setup_requires` section.